cypress.config.js

```diff
@@ -67,7 +66,8 @@ module.exports = defineConfig({
 			if (semver.satisfies(config.env.phpSemverVersion, '<7.3.0')) {
 				config.excludeSpecPattern = config.excludeSpecPattern.concat(
 					[
-						"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/steps/ecommerce/**",
+						"tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/2-ecommerce/**",
```

Comment:
Do we really need `1-`, `2-` before for the top level directories? I am seeing them execute in a random order in the workflow run

Reply:
Only one order is not working as expected i.e the last step is running before intermediate step.

Reply:
we are using 1- , 2- to run in specific order.